install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2845,6 +2885,12 @@ clustermesh:
       # -- Configure the port the apiserver metric server listens on.
       port: 9962
 
+      kvstoremesh:
+        # -- Enables exporting KVStoreMesh metrics in OpenMetrics format.
+        enabled: false
```

Comment:
Can this be enabled by default? Since kvstoremesh isn't a host-network pod, it shouldn't cause any issues.

Reply:
I personally don't have a strong opinion here. ATM, metrics are disabled by default for consistency with the clustermesh-apiserver/etcd ones, but we could also switch those to be enabled by default in case.

Reply:
I've switched them to be enabled by default both for the clustermesh-apiserver and kvstoremesh.